bin/damsolrizer

```diff
@@ -101,12 +101,23 @@ begin
       max_tries = 2
       puts @msg.headers.inspect
       puts "PID: #{@msg.headers["pid"]}"
+      puts "method: #{method}"
       begin
-        if ["addDatastream", "addRelationship","ingest","modifyDatastreamByValue","modifyDatastreamByReference","modifyObject","purgeDatastream","purgeRelationship"].include? method
+        if ["modifyCollection", "addDatastream", "addRelationship","ingest","modifyDatastreamByValue","modifyDatastreamByReference","modifyObject","purgeDatastream","purgeRelationship"].include? method
           solrizer = Solrizer::Fedora::Solrizer.new
           solrizer.solrize @msg.headers["pid"], solropts
           puts  "updated solr index for #{@msg.headers["pid"]}\n"
-        elsif method == "purgeObject"
+          if method == "modifyCollection"
+            msg = Dams::ShareApi.share_push(pid)
```

Comment:
Will this push all collections with type "modifyCollection" to OSF? Do we need to check whether a collection is in OSF or not first before pushing the share api update?